make/modules/java.base/Copy.gmk

```diff
@@ -118,6 +117,26 @@ $(JVMCFG): $(call DependOnVariable, ORDERED_CFG_VARIANTS)
 
 TARGETS += $(JVMCFG)
 
+################################################################################
+# Copy import JVMs into place, if any
+#
+ifneq ($(IMPORT_JVM_NAMES),)
+  rename_to_libjvm = $(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
+
+  $(foreach import_jvm, $(IMPORT_JVM_NAMES), \
+    $(eval $(import_jvm)_path := $(patsubst $(import_jvm):%,%,$(filter $(import_jvm):%, $(IMPORT_JVMS)))) \
+    $(eval $(call MakeDir, $(LIB_DST_DIR)/$(import_jvm))) \
```

Comment:
Do you really need to call MakeDir outside of a recipe? Won't SetupCopyFiles recipes create this directory?

Reply:
I assumed it would, but it failed, and I looked at the code and found no clear evidence that it does, so I added this. It might have been some other issue, so I can double-check again, if you are certain that SetupCopyFiles should create needed directories.